controller/v1/base/home.go

```diff
@@ -0,0 +1,17 @@
+package base
+
+import (
+	"net/http"
+
+	"github.com/opensaucerer/gotemplate/helper"
+	"github.com/opensaucerer/gotemplate/typing"
+)
+
+func Home(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is called home.... why can we see the word `health` inside it?

```go
helper.SendJSONResponse(w, true, http.StatusOK, "Home", typing.M{"home": data})
```

Reply:
Now I really have to pay more attention to making sure my updates are all intact in github. The `RegisterHealthCheckRoute` you told me to change has initially been changed but the update isn't visible here. I don't know why I keep having issues with git, I will make sure i pay attention to my updates next time.